docs/serverless/03-03-registries.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Internal and external registries
+type: Details
+---
+
+In its default configuration, Serverless uses persistent volumes as the internal registry to store Docker images for Functions. The default storage size of a single volume is 20 GB. This internal registry is suitable for local development.
+
+If you use Serverless for production purposes, it is recommended that you use an external registry, such as Docker Hub, Google Container Registry (GCR), or Azure Container Registry (ACR).
+
+Serverless supports two ways of connecting to an external registry:
+
+- [You can set up an external registry before installation](#tutorials-set-an-external-docker-registry).
+
+  In this scenario, you can use Kyma overrides to change the default values supplied by the installation mechanism.
+
+- [You can switch to an external registry in the runtime](#tutorials-switch-to-an-external-docker-registry).
+
+  In this scenario, you can change the registry on the fly, with Kyma already installed on your cluster. This option gives you way more flexibility and control over the choice of an external registry for your Functions.
+
+## Switching registries in the runtime
+
+When you install Kyma with the default internal registry, Helm creates the `serverless-registry-config-default` Secret in the `kyma-system` Namespace. This Secret contains credentials used by Kubernetes to pull deployed Functions' images from the internal registry. They also allow Kaniko to push any images to the registry each time a Function is created or updated.
```

Comment:
```suggestion
When you install Kyma with the default internal registry, Helm creates the `serverless-registry-config-default` Secret in the `kyma-system` Namespace. This Secret contains credentials used by Kubernetes to pull deployed Functions' images from the internal registry. It also allows Kaniko to push any images to the registry each time a Function is created or updated.
```
I think the third sentence refers to "this secret"?

Reply:
I meant `credentials` but maybe I will just repeat the word to make it clearer :)